### PR TITLE
Add file explorer to pack-interact workflow

### PR DESCRIPTION
**Pull request by aemengo**
## Summary
Adds a _dive_-style container layers explorer to `--interactive` mode, but delineated by buildpack

## Output
Screen Shot 2021-11-10 at 10 47 54 PM

## Documentation

- Should this change be documented?
    - [ ] Yes, see #___
    - [ ] No
    - [x] At some point

## Related

Resolves #___

### CODE REVIEW

File: internal/termui/detect.go [hunk 1]
```diff
@@ -46,6 +46,10 @@ func (d *Detect) Stop() {
 	d.doneChan <- true
 }
 
+func (d *Detect) SetNodes(map[string]*tview.TreeNode) {
+	// not implemented
```
Comment (dfreilich) [inline internal/termui/detect.go:50]: What will be happening here?

**Review by dfreilich — APPROVED** [review]
From a UX perspective, this is fabulous. I really appreciated how, even before you added in a toolbar for what commands are valid, that the colors of the buildpacks changed when you were able to select them. I thought that was a very nice signifier.

**Comment by aemengo** [comment]
@dfreilich You still had good feedback about showing a keymap - at least for **Ctrl-C**. I haven't forgotten; it will be addressed in a subsequent PR.

File: internal/termui/detect.go (hunk 1 above)
Comment (aemengo) [reply internal/termui/detect.go:50]: Unfortunately that was me simply trying to satisfy the `type page interface`. That method is never intended to be implemented, but the _other page_ (Dashboard) does have it implemented.

Overall, I have serious misgivings the page orchestration. I planned to refactor but simply didn't come up with something I liked. If you'd prefer, I can spruce it up relatively soon.

```
type page interface {
	Handle(txt string)
	Stop()
	SetNodes(nodes map[string]*tview.TreeNode)
}
```

Comment (dfreilich) [reply internal/termui/detect.go:50]: Hmmm. I think it's ok to merge it in without it spruced up, given how it's hidden within an experimental flag and is going to be worked on more, but maybe add that in in a clarifying comment?

Comment (aemengo) [reply internal/termui/detect.go]: I added clarifying comments.

**Pull request merged**